### ISSUE TITLE
Transition drawers are not indented correctly.

### ISSUE THREAD

**Issue by Setmaster** [KybernetikGames/animancer#246]
# Environment

- Animancer Version Number: 7.4a2
- Animancer Pro or Lite: Pro
- Unity Version: 2022.2.2f1
- Platform: Windows 10

# Description

Animancer drawers are not shown correctly. The expand button is rendered behind the field name.

# Reproduction

Steps to reproduce the bug:

1. Create new Unity 2022.2.2f1 project
2. Import animancer
3. Open example scene such as layers
4. See the issue in the inspector

https://user-images.githubusercontent.com/5375992/213386905-40164ed8-ada0-4cca-80f1-fca6471ce42f.mp4

**Comment by KybernetikGames** [comment]
Looks like they changed the way indentation works. For example:

![image](https://user-images.githubusercontent.com/43698611/213420130-c393a299-b0a4-4dd5-a2a4-5aef7e6bad59.png)

Unity 2021.3 draws this:

![image](https://user-images.githubusercontent.com/43698611/213420275-017b8834-675c-4acc-a742-12b876f6b3da.png)

And 2022.2 draws this:

![image](https://user-images.githubusercontent.com/43698611/213420404-0d5677ae-f5eb-4598-85e5-9821884ef98c.png)

I like the old look better and I can adjust things to go back to like they were, but for whatever reason that ends up breaking the foldout arrow. It still lets you expand/collapse by clicking on the label but clicking the arrow itself does nothing if it's at indent level 0 (but works fine at any other level). So I just gave up and went with their new style.

I've uploaded the fix to the same link you downloaded v7.4a2 from.
